spec/models/course_spec.rb

```diff
@@ -1,21 +1,26 @@
 require 'rails_helper'
 
 RSpec.describe Course, type: :model do
+  
   subject { course }
   context 'with valid attributes' do
-    let(:course) { build(:course, name: "course1", description: "description", number_of_semesters: 1, start_date: "2021-01-20", end_date: "2020-05-20", allocation: 10) }
+     let!(:course) { build(:course, name: "course1", description: "description", number_of_semesters: 1, start_date: "2021-01-20", end_date: "2020-05-20", allocation: 10) }
```

Comment:
Minor: Can you remove the additional space here?
If you're using VS Code, I can recommend the [indent-rainbow extension](https://marketplace.visualstudio.com/items?itemName=oderwat.indent-rainbow).

It makes it much easier to see indentation levels and highlights any lines where you've got an odd number of leading spaces.

![Screen Shot 2021-03-23 at 10 20 41 am](https://user-images.githubusercontent.com/289330/112072345-8fa77f00-8bc1-11eb-98ad-617eb6bbc5d8.png)

I'm sure other editors have something similar.